Makefile

```diff
@@ -103,3 +104,6 @@ test_one: clean
 
 js_external:
 	$(MAKE) -C static/js/external
+
+static_external:
+	$(MAKE) -C static/external
```

Comment:
JS files are static files, so it overlaps to have both `js_external` and `static_external`. Maybe only call `static_external` from here, and then call `js_external` from `static_external`?